### PR TITLE
Promote GM new addressprefix and dapp

### PR DESCRIPTION
**Pull request by leohar**

### CODE REVIEW

File: dapps/dapps.json [hunk 1]
```diff
@@ -467,6 +467,14 @@
       "categories": [
         "defi"
       ]
+    },
+    {
+      "name": "GM - Say it back!",
```
Comment (antonkhvorov) [inline dapps/dapps.json:472]: GM! Say it back

File: dapps/dapps_dev.json [hunk 2]
```diff
@@ -588,7 +588,7 @@
       ]
     },
     {
-      "name": "GM OR DIE",
+      "name": "GM - Say it back!",
```
Comment (antonkhvorov) [inline dapps/dapps_dev.json:591]: GM! Say it back

**Review by antonkhvorov — CHANGES_REQUESTED** [review]

File: dapps/dapps.json (hunk 1 above)
Comment (leohar) [reply dapps/dapps.json]: done

File: dapps/dapps_dev.json (hunk 2 above)
Comment (leohar) [reply dapps/dapps_dev.json]: done

**Review by antonkhvorov — APPROVED** [review]

**Review by ERussel — APPROVED** [review]

**Pull request merged**